src/diet.ts

```diff
@@ -477,76 +501,14 @@ function itemPriority(menuItems: MenuItem[]) {
   }
 }
 
-export function runDiet(): void {
-  pillCheck();
-
-  nonOrganAdventures();
-
-  if (myFamiliar() === $familiar`Stooper`) {
-    useFamiliar($familiar`none`);
-  }
-
-  if (have($item`astral six-pack`)) {
-    use($item`astral six-pack`);
-  }
-
-  fillShotglass();
-
-  if (
-    get("barrelShrineUnlocked") &&
-    !get("_barrelPrayer") &&
-    $classes`Turtle Tamer, Accordion Thief`.includes(myClass())
-  ) {
-    cliExecute("barrelprayer buff");
-  }
-
-  const { bestSpleenItem } = getBestSpleenItems();
-  const embezzlers = embezzlerCount();
-  if (embezzlers) {
-    if (
-      mySpleenUse() + 2 <= spleenLimit() &&
-      !get("beGregariousCharges") &&
-      !(
-        get("beGregariousFightsLeft") &&
-        get("beGregariousMonster") === $monster`Knob Goblin Embezzler`
-      )
-    ) {
-      const value = expectedGregs() * 25000;
-      if (
-        value - mallPrice($item`Extrovermectin™`) >
-        5 * MPA - 2 * mallPrice($item`transdermal smoke patch`)
-      ) {
-        acquire(1, $item`Extrovermectin™`, value);
-        chew(1, $item`Extrovermectin™`);
-      }
-    }
-    if (mySpleenUse() < spleenLimit()) {
-      if (!have($effect`Eau d' Clochard`)) {
-        if (!have($item`beggin' cologne`)) {
-          const cologne = new Potion($item`beggin' cologne`);
-          const equilibriumPrice = cologne.gross(embezzlers) - valuePerSpleen(bestSpleenItem);
-          if (equilibriumPrice > 0) buy(1, $item`beggin' cologne`, equilibriumPrice);
-        }
-        if (have($item`beggin' cologne`)) {
-          chew(1, $item`beggin' cologne`);
-        }
-      }
-    }
-    if (have($item`body spradium`) && mySpleenUse() < spleenLimit()) {
-      ensureEffect($effect`Boxing Day Glow`);
-    }
-  }
-
-  const diet = computeDiet();
-
-  // Sort descending by item priority.
-  diet.sort(([x], [y]) => -(itemPriority(x) - itemPriority(y)));
-
+export function consumeDiet(diet: [MenuItem[], number][]): void {
   print();
   print("===== PLANNED DIET =====");
   printDiet(diet);
   print();
 
+  diet.sort(([x], [y]) => -(itemPriority(x) - itemPriority(y)));
```

Comment:
```suggestion
  diet.sort(([a], [b]) => itemPriority(b) - itemPriority(a));
```

This is more consistent with other writing we see in garbo